Fitter/python/plot/datacard.py

```diff
@@ -246,15 +247,15 @@ def stackinputs(file,variable,processes,**kwargs):
     hist.SetDirectory(0)
     hist.SetLineColor(kBlack)
     hist.SetFillStyle(1001) # assume fill color is already correct
-    if process=='data_obs':
+    if 'data_obs' in process:
```

Comment:
What's the reason for this change, btw?

Reply:
Actually, I was asked to compare the ZTT distributions only with data_obs where data_obs is the data_obs after subtracting all non ZTT distribution. And I saved that data_obs with name 'data_obs_nonztt_subtratced'. That's why I made those changes and in the list of varprocs I put "data_obs_nonztt_subtratced" instead of "data_obs".

Reply:
Okay, I see. Let's just leave it for now.